models/src/it/java/io/sphere/sdk/products/ProductProjectionSearchIntegrationTest.java

```diff
@@ -46,10 +46,10 @@
     @BeforeClass
     public static void setupProducts() {
         removeProducts();
-        final TextAttributeDefinition colorAttributeDefinition = TextAttributeDefinitionBuilder
-                .of(COLOR, LocalizedStrings.ofEnglishLocale(COLOR), TextInputHint.SINGLE_LINE).build();
-        final TextAttributeDefinition sizeAttributeDefinition = TextAttributeDefinitionBuilder
-                .of(SIZE, LocalizedStrings.ofEnglishLocale(SIZE), TextInputHint.SINGLE_LINE).build();
+        final AttributeDefinition colorAttributeDefinition = AttributeDefinitionBuilder
+                .of(COLOR, LocalizedStrings.ofEnglishLocale(COLOR), new TextType()).build();
```

Comment:
Do you mean for `TextType`? You told me that this area is restored in some other branch heavily so does it make sense to do it in this branch?